builtin/rebase--helper.c

```diff
@@ -44,6 +47,25 @@ static int get_revision_ranges(const char *upstream, const char *onto,
 	return 0;
 }
 
+static int init_basic_state(struct replay_opts *opts, const char *head_name,
+			    const char *onto, const char *orig_head)
+{
+	struct stat sb;
+	FILE *interactive;
+
+	if (stat(path_state_dir(), &sb) && mkdir_in_gitdir(path_state_dir()))
+		return error(_("could not create temporary %s"), path_state_dir());
```

Comment:
Maybe using error_errno() would be better as I am not sure that mkdir_in_gitdir() would output an error message.